apps/marketing/src/components/(marketing)/widget.tsx

```diff
@@ -391,6 +391,7 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
 
           <SignaturePad
             className="aspect-video w-full rounded-md border"
+            defaultValue={draftSignatureDataUrl || ''}
```

Comment:
Shouldn't this be `signatureDataUrl || ''` instead?

Assuming that we should only persist signatures that they explicitly click 'confirm' on

Reply:
You are correct. Otherwise, there's also a mismatch between the modal and the preview. Pushing an update now.